x-pack/plugins/fleet/public/applications/fleet/types/ui_extensions.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ComponentType, LazyExoticComponent } from 'react';
+import { NewPackagePolicy, PackagePolicy } from '../../../../common/types/models';
```

Comment:
you should be able to import these types from this directory (`./index` or `./`)